packages/flutter_tools/test/general.shard/resident_web_runner_test.dart

```diff
@@ -189,6 +189,8 @@ void main() {
       'cd29': 'false',
       'cd30': 'true',
     })).called(1);
+    verify(Usage.instance.sendTiming('hot', 'web-restart', any)).called(1);
```

Comment:
Should there have been a web-recompile here too?

Reply:
Done